src/components/organisms/ConversationWindow.vue

```diff
@@ -110,6 +120,21 @@ export default {
         ? null
         : messages[messages.length - 1].suggestedActions?.actions;
     });
+    const messageTimestamp = computed(() => {
+      const msgs = chatMessage.value.messages;
+      if (msgs?.length >= 1) {
+        //create a timestamp using the oldest message sent by the bot
+        return new Date(msgs[0].timestamp).toLocaleString(
+          useI18n().locale.value,
+          {
+            weekday: "short",
+            hour: "2-digit",
+            minute: "2-digit",
+          }
+        );
+      }
+      return ""; //return nothing if no messages exist
```

Comment:
Instead of returning an empty string, we could return a new Date.now. this way we can remove our hard-coded messageTime locale. 

We can just leave it as an empty string. Up to you :)

Reply:
Do you think that something like "Message received date not available" or "The date the message was received is not available" might be more suitable?

Reply:
I would not but I believe that would be a question for Thomas Hoy.

Reply:
On second thought, decided to go for using the current time as it makes more sense.